### PR TITLE
Project page changed

### PR DESCRIPTION
**Pull request by Adyasha8105**
- I have added an animation effect in the header.
- I have changed the card style (with hover effect)  and made it responsive also.
- I have added the SVG image (dummy).

@anushbhatia @smaranjitghose Please check it :).

### CODE REVIEW

**Comment by shubhkhanna** [comment]
@smaranjitghose @anushbhatia sir, please look into the matter. It's a huge request that every participant first focus on there page and do not make any changes to other pages...if some issue is present in the page then create another issue....@Adyasha8105 you are making changes to animation timing & colors, that's not good & not providing relatable title for PR.

We can test these things in test branch after page is fully created.

**Comment by Adyasha8105** [comment]
@shubhkhanna Chill yaar. Its a matter of single background color(single-line code). If the mentor doesn't like this white background in education page color then I will revert it back. Don't create an issue we can resolve it easily.

**Comment by sakshi1499** [comment]
@Adyasha8105 @shubhkhanna This isn't a way to talk, please both of you be patient. We are in an open source Community where all of us should work together to make things better. This isn't a fight between anything. Few months down the line, when this project will be ready to serve, you both will be able to say that we worked on it together and these matters are really small, please don't ruin the spirit of Open source.
Secondly, please don't take up issues that are assigned to others or others are working on. But if they are done working on it, anybody, not just you or me, BUT ANYBODY, can come and work on it.
Please don't use slang, lets be professional please.

**Comment by Adyasha8105** [comment]
@sakshi1499 @shubhkhanna I have removed all those changes done by me in the Education page :)

File: projects.html [hunk 1]
```diff
@@ -41,100 +42,73 @@
...
-        in your favor"
+<div class="container" style="width: 100%;">
+<section style=" height: 70vh;" class="animate__animated animate__fadeInDown">
+  <div class="px-3 pt-5">
+    <div class="row text-dark">
+      <div
+        class="col-md-6 d-flex mt-5 flex-column text-center justify-content-center animate__animated animate__fadeInLeft animate__delay-0.5s">
+        <h2 style="font-size: 4em;font-weight: bolder;" class="mb-0">Projects</h2>
+        <p class="font-weight-bolder text-muted" style="font-size: 1.2em; letter-spacing: 2px;">
+          My projects makes use of vast variety of latest technology tools. My best experience is to create Data Science projects and deploy them to web applications using cloud infrastructure.
```
Comment (sakshi1499) [inline projects.html:53]: I feel the text is very clumsy, Align it on one side or ask @anushbhatia for the same, I am not an expert at Design. But it looks clumsy.

File: projects.html [hunk 2]
```diff
@@ -41,100 +42,73 @@
...
-            </li>
-            <li>
-              <a href="#"><i class="fa fa-medium"></i></a>
-            </li>
-          </ul>
+      <div class="col-md-4">
+          <div class="skill-card">
+            <header class="skill-card__header"><img class="skill-card__icon" src="./assets/images/project-page/project-card.svg"></header>
+            <section class="skill-card__body">
+                <h2 class="skill-card__title">Project_Name</h2><span class="skill-card__duration">Hover me</span>
```
[the remark this answers is not in the document]
Comment (Adyasha8105) [reply projects.html:84]: Actually its a dummy one.

File: projects.html (hunk 1 above)
Comment (Adyasha8105) [reply projects.html:53]: okay then :)

**Comment by Adyasha8105** [comment]
@sakshi1499 Is it looks fine now? about the text.

**Pull request merged**